### PR TITLE
New Ionic docs

### PR DESCRIPTION
**Pull request by lobo**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/native-platforms/ionic/04-linking-accounts.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+
+}());
+
+```
+
+## User Profile Linked Accounts Information
+
+The user profile contains an array of identities which includes the profile information from linked providers. 
+
+To view a user's identities, access the [Users](${uiURL}/#/users) page on the Auth0 dashboard, select a user, and scroll down to `identities`.
```
Comment (mpaktiti) [inline articles/native-platforms/ionic/04-linking-accounts.md:146]: @lobo `uiURL` is not used any longer, so i throws `ReferenceError: uiURL is not defined`. Replace it with `manage_url`.

File: articles/native-platforms/ionic/05-rules.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  githubUrl: 'https://github.com/auth0-samples/auth0-ionic-samples',
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-ionic-samples',
+  pkgBranch: 'master',
+  pkgPath: '05-Rules',
+  pkgFilePath: null,
+  pkgType: 'replace'
+}) %>
+
+<%= include('../../_includes/server-platform/_rules-introduction') %>
```
Comment (mpaktiti) [inline articles/native-platforms/ionic/05-rules.md:16]: @lobo Throws `Error: Could not find include ../../_includes/server-platform/_rules-introduction`. There is no such directory.

File: articles/native-platforms/ionic/05-rules.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Create a Rule
+
+<%= include('../../_includes/server-platform/_rules-create-section') %>
```
Comment (mpaktiti) [inline articles/native-platforms/ionic/05-rules.md:20]: @lobo There is no such directory.

File: articles/native-platforms/ionic/05-rules.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+## Test the Rule
+
+<%= include('../../_includes/server-platform/_rules-test-result-intro', { profilelink: '/docs/quickstart/native/ionic/03-user-profile' }) %>
```
Comment (mpaktiti) [inline articles/native-platforms/ionic/05-rules.md:24]: @lobo There is no such directory.

File: articles/native-platforms/ionic/06-authorization.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+  githubUrl: 'https://github.com/auth0-samples/auth0-ionic-samples',
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-ionic-samples',
+  pkgBranch: 'master',
+  pkgPath: '06-Authorization',
+  pkgFilePath: null,
+  pkgType: 'replace'
+}) %>
+
+<%= include('../_includes/_authorization-introduction', { ruleslink: '/docs/quickstart/native/ionic/05-rules' }) %>
```
Comment (mpaktiti) [inline articles/native-platforms/ionic/06-authorization.md:16]: @lobo Throws `Error: Could not find include ../_includes/_authorization-introduction`. There is no such directory.

File: articles/native-platforms/ionic/06-authorization.md [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+
+### Create a Rule to assign roles
+
+<%= include('../_includes/_authorization-create-rule') %>_
```
Comment (mpaktiti) [inline articles/native-platforms/ionic/06-authorization.md:20]: @lobo There is no such directory.

File: articles/native-platforms/ionic/08-mfa.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+  githubUrl: 'https://github.com/auth0-samples/auth0-ionic-samples',
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-ionic-samples',
+  pkgBranch: 'master',
+  pkgPath: '08-MFA',
+  pkgFilePath: null,
+  pkgType: 'replace'
+}) %>
+
+<%= include('../_includes/_mfa-introduction') %>
```
Comment (mpaktiti) [inline articles/native-platforms/ionic/08-mfa.md:16]: @lobo Throws `Error: Could not find include ../_includes/_mfa-introduction`. There is no such directory.

File: articles/native-platforms/ionic/08-mfa.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+In this tutorial, you will learn how to enable MFA in the Ionic application you created in the previous steps.
+
+## Enable Multifactor Authentication in Your Account
+
+<%= include('../_includes/_mfa-enable') %>
```
Comment (mpaktiti) [inline articles/native-platforms/ionic/08-mfa.md:22]: @lobo There is no such directory.

File: articles/native-platforms/ionic/08-mfa.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+## Login
+
+<%= include('../_includes/_mfa-login', { loginlink: '/docs/quickstart/native/ionic/01-login' }) %>
```
Comment (mpaktiti) [inline articles/native-platforms/ionic/08-mfa.md:26]: @lobo There is no such directory.

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Hey @lobo ! Most errors are due to missing files. Please review my comments, apply any changes needed and let me know if you need anything else. Keep in mind that most of the times heroku is not very useful at troubleshooting, I find deploying the site locally much more helpful.

**Pull request closed without merging**